main.py

```diff
@@ -1,17 +1,121 @@
 #!/usr/bin/env python3
 ############################################################################
 # Soubor:  main.py
-# Datum:
-# Autor:
+# Datum: 10.1.2022
+# Autor: Michal Vojtěchovský
 ############################################################################
-from random import randint, choice
+from random import randint, choice, random
+import time
 
 ############################################################################
 
+def lowerCaseConvertor(file):
+    try:
+        f = open(f"{file}.txt","r")
+    except FileNotFoundError:
+        print("soubor nenalezen")
+        menu()    
```

Comment:
Tohle není dobrý nápad: poprvé to zadáte špatně, program se zanoří, pak to zadáte dobře, provede se korektně akce, ale potom se opět vynoříte a program spadne